supersid/supersid_versions.py

```diff
@@ -36,11 +36,14 @@
     'cycler': {'versions': ['0.10.0', '0.11.0'], 'requirement': 'mandatory'},
     'six': {'versions': ['1.12.0', '1.16.0'], 'requirement': 'mandatory'},
     'kiwisolver': {'versions': ['1.3.1', '1.3.2'], 'requirement': 'mandatory'},
-    'numpy': {'versions': ['1.21.2', '1.21.3', '1.21.4+vanilla'], 'requirement': 'mandatory'},
+    'numpy': {'versions': ['1.21.2', '1.21.3', '1.21.4+vanilla', '1.21.5'], 'requirement': 'mandatory'},
     'pillow': {'versions': ['8.4.0'], 'requirement': 'mandatory'},
     'pyparsing': {'versions': ['2.4.7', '3.0.4'], 'requirement': 'mandatory'},
     'python-dateutil': {'versions': ['2.8.2'], 'requirement': 'mandatory'},
 
+    # pandas dependencies
+    'pandas': {'versions': ['1.3.5'], 'requirement': 'mandatory'},
```

Comment:
'1.3.5' is not available in the conda installation on Windows, thus the accepted versions should be ['1.3.4', '1.3.5'].

Reply:
Added pandas 1.3.4 to acceptable versions.